app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyDetailScreen.kt

```diff
@@ -73,6 +84,7 @@ fun SurveyDetailScreen(
     val currentPage by viewModel.currentPage.collectAsState()
     val surveyQuestions by viewModel.surveyQuestions.collectAsState()
     val shouldShowLoading by viewModel.shouldShowLoading.collectAsState()
+    val shouldShowLottie by viewModel.shouldShowLottie.collectAsState()
```

Comment:
How about renaming this property to something more abstract? e.g., `shouldShowThanks`/`shouldShowSuccessfulSubmission`/ `shouldCelebrate` 😄 

Think about the case where we might want to replace Lottie with some other library for this animation, which should be dealt with only in this class, not in the VM.